pages/samvera/developer_community/how-to-pr.md

```diff
@@ -8,3 +8,24 @@ folder: samvera/community/
 sidebar: home_sidebar
 tags: [community]
 ---
+## How do I contribute my changes to the project?
+
+Generally speaking, we follow the guidelines / recommendations in [Hydra Technical On-Boarding](https://github.com/escowles/technical-onboarding/blob/master/code.md) regarding both Code Guidelines and Code Reviews.
```

Comment:
Since we are no longer "Hydra", I'd suggest the title you show here just say "Technical On-boarding".